test/counter.test.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable linebreak-style */
+
+import elementsCounter from '../src/modules/counter';
+
```

Comment:
- [ ] <b>REQUIRED:</b> You tests are failing. Kindly fix them.
      ![Screen Shot 2021-10-16 at 16 49 11](https://user-images.githubusercontent.com/45217364/137592008-0bb81597-886b-4442-b075-452e289cf34b.png)

Reply:
Thanks! It's done.  I cloned the project in another folder and of course, the tests didn't work.  The dev branch was missing .babelrc because of .gitignore.  Now they work.